code/modules/requisitions/supply_export.dm

```diff
@@ -33,7 +37,120 @@
 		if(JOB_CAT_COMMAND)
 			. = 100
 	SSpoints.supply_points[faction_selling] += .
-	return new /datum/export_report(., name, faction_selling)
+	SSpoints.export_history += new /datum/export_report(., name, faction_selling)
+
+/**Calculates return value of properly prepared corpses, handles point payout and export logging.
+ *
+ * Checks for:
+ **Five minute defib timer
+ **Any brute or burn damage, including internal bleeding
+ **Any limbs requiring repair, bearing an open surgical incision, or carrying high germs
+ **Eye damage or disfigurement
+ **Missing synth head, on top of the normal missing limb penalty
+ **Shrapnel
+ **Full dress uniform: torso, hat, gloves, boots
+ **Suicide/export before shutters open
+ **Dogtag in memorial, if they've got one
+ */
+/mob/living/carbon/human/proc/burial_export(faction_selling)
+	var/report_name = name
+	var/list/problems = list()
+	var/payout = 0
+
+	switch(job.job_category)
+		if(JOB_CAT_ENGINEERING || JOB_CAT_MEDICAL || JOB_CAT_REQUISITIONS)
+			payout = 5	//Shipside crew shouldn't be dying in any situation where you can still use requisitions.
+		if(JOB_CAT_MARINE)
+			payout = 20
+		if(JOB_CAT_SILICON)
+			payout = 50	//Silicons can get recycled so they're worth extra
+		if(JOB_CAT_COMMAND)
+			payout = 30
+
+	if(!HAS_TRAIT(src, TRAIT_UNDEFIBBABLE))
+		problems += "lingering vitals"
+		payout = 0
```

Comment:
return ? Also, synth never have that trait IIRC

Reply:
I also think that req lift should not work if someone is revivable

Reply:
Synths get it normally, they just ignore it while getting defibbed. Making it just not lowerable without is probably better, yeah.